src/Service.Tests/DatabaseSchema-MsSql.sql

```diff
@@ -82,7 +82,7 @@ CREATE TABLE publishers_mm(
 
 CREATE TABLE books(
     id int IDENTITY(5001, 1) PRIMARY KEY,
-    title varchar(max) NOT NULL,
+    title varchar(30) NOT NULL,
```

Comment:
Should we test something similar for PostgreSQL/MySQL?

Reply:
That would be a good idea, but I don't have those setup to validate.